packages/tessellate-request/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "tessellate-request",
+  "version": "0.0.1",
+  "description": "Tessellate request library.",
+  "author": "Sebastian Monte <sebastian.monte@zalando.de>",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "publishConfig": {
+    "tag": "next"
+  },
+  "engines": {
+    "node": ">=6"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/zalando-incubator/tessellate.git"
+  },
+  "scripts": {
+    "dist": "webpack",
+    "lint": "eslint .",
+    "test": "NODE_ENV=test jest --coverage"
+  },
+  "dependencies": {
+    "client-oauth2": "3.2.0",
+    "nconf": "0.8.4",
+    "prom-client": "6.1.2",
+    "prometheus-gc-stats": "0.3.1",
```

Comment:
We don't need prom-client and prometheus-gc-stats (they should used in tessellate-server only).